EIPS/eip-5732.md

```diff
@@ -0,0 +1,106 @@
+---
+eip: 5732
+title: Simple Commit Interface
+description: A simple but general commit interface to support commit-reveal scheme.
```

Comment:
```suggestion
description: A simple generic interface to support a commit-reveal scheme.
```
How about this?

Reply:
haha just wanna use the "but" to separate "simple" and "general" because they refer to different charecteristics of this EIP